apps/amplify-demo/.gitignore

```diff
@@ -37,4 +38,25 @@ amplifyconfiguration.json
 amplify-build-config.json
 amplify-gradle-config.json
 amplifytools.xcconfig
-.secret-*
\ No newline at end of file
+.secret-*
+
+#amplify-do-not-edit-begin
+amplify/\#current-cloud-backend
+amplify/.config/local-*
+amplify/logs
+amplify/mock-data
+amplify/backend/amplify-meta.json
+amplify/backend/.temp
+build/
+dist/
+node_modules/
```

Comment:
1. `build/` and `/dist` are duplicate.
2. Not sure if we should put `node_modules/` in this `#amplify-do-not-edit` section.

Reply:
the amplify-do-not-edit section is automatically put there by amplify themselves. We should not modify it.